src/views/job invites.ejs

```diff
@@ -0,0 +1,99 @@
+<% include partials/nav %>
+    <% include partials/header %>
+
+        <body>
```

Comment:
Why do we have this file name as `job invites`? It violates our variable naming convention and even has space in it. Kindly use p`ascalNameing`